cpp-audio-studio/web/plugin-pro-app.js

```
@@ -0,0 +1,186 @@
+    // Get peak and RMS
+    const peak = engine ? Math.min(1.0, engine.getPeakLevel()) : 0;
+    const rms = engine ? Math.min(1.0, engine.getRMSLevel()) : 0;
+    
+    // Draw entire UI using C++
+    pluginUI.drawUI(peak, rms, waveform, spectrum);
+    
+    // Execute all draw commands from C++
+    const commands = pluginUI.createDrawCommands();
+    for (let i = 0; i < commands.length; i++) {
+        try {
+            eval(commands[i]);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)